src/idiomatic/leveraging-the-type-system/newtype-pattern.md

```diff
@@ -0,0 +1,49 @@
+---
+minutes: 5
+---
+
+# Newtype Pattern
+
+A _newtype_ is a wrapper around an existing type, often a primitive:
```

Comment:
I would like a throw-back to the other page that introduces the newtype pattern (perhaps in the speaker notes, depending on how this looks on screen).

Reply:
I've added bidirectional links in 51d050eb.